src/test/java/bio/terra/service/snapshot/SnapshotPermissionsIntegrationTest.java

```diff
@@ -123,6 +124,10 @@ public void tearDown() throws Exception {
   void snapshotInvalidEmailTest() throws Exception {
     SnapshotRequestModel requestModel =
         jsonLoader.loadObject("ingest-test-snapshot.json", SnapshotRequestModel.class);
+    // randomize name so that we don't have failures on test retry after successful creation of snapshot
+    // Only randomize name here at the beginning of the test
+    // so that we can test that we can't create a snapshot with the same name
+    requestModel.setName(Names.randomizeName(requestModel.getName()));
```

Comment:
This is not related to my other changes, but ran into a test failure for this test during my test runs: https://scans.gradle.com/s/64jc6uumax4su/tests/task/:testIntegration/details/bio.terra.service.snapshot.SnapshotPermissionsIntegrationTest/snapshotInvalidEmailTest()?top-execution=1